src/domain/common/license-entitlement/license.entitlement.service.ts

```
@@ -0,0 +1,193 @@
+    }
+
+    const entitlementLimit = licenseEntitlement.limit;
+    let entitlementsUsed = 999;
+    switch (license.type) {
+      case LicenseType.ACCOUNT:
+        entitlementsUsed = await this.getEntitlementUsageUsingEntities(
+          license,
+          licenseEntitlement
+        );
+        break;
+      case LicenseType.ACCOUNT:
```

Biome

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)